.github/workflows/run-merge-hive-tests.yml

```diff
@@ -73,63 +51,63 @@ jobs:
+        uses: actions/checkout@v3
       - name: Unshallow fetching
         run: git fetch --unshallow
-      - name: Setting up QEMU
+      - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
-      - name: Setting up Docker Buildx
+      - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
-      - name: Building Nethermind Image
+      - name: Build Nethermind Docker image
         uses: docker/build-push-action@v2
         with: 
           context: .
           file: ./Dockerfile
           tags: nethermind:test-${{ github.sha }}
           outputs: type=docker,dest=/tmp/image.tar
-      - name: Installing Linux packages
+      - name: Install Linux packages
         run: |
-         sudo apt-get update
-         sudo apt-get install libsnappy-dev libc6-dev libc6 build-essential
-      - name: Setup Go environment
+          sudo apt-get update
+          sudo apt-get install libsnappy-dev libc6-dev libc6 build-essential
+      - name: Set up Go environment
         uses: actions/setup-go@v3.0.0
         with:
-         go-version: '>=1.17.0'
-      - name: Check out Hive
-        uses: actions/checkout@v2
+          go-version: '>=1.17.0'
+      - name: Check out Hive repository
+        uses: actions/checkout@v3
         with:
           repository: ${{ github.event.inputs.hive }}
           ref: ${{ github.event.inputs.branch }}
       - name: Change Hive configuration
         run: sed -i 's#FROM nethermindeth/hive:$branch#FROM nethermind:test-${{ github.sha }}#g' clients/nethermind/Dockerfile
       - name: Build Hive
         run: go build .
-      - name: Load Docker Image
+      - name: Load Docker image
         run: docker load --input /tmp/image.tar
       - name: Run Hive
         continue-on-error: true
         if: "${{ github.event.inputs.limit == '' }}"
         run: ./hive --client nethermind --sim ${{ github.event.inputs.testSuite }} --sim.loglevel ${{ github.event.inputs.loglevel }}
-      - name: Run Hive With Limit
+      - name: Run Hive with limit
         continue-on-error: true
         if: "${{ github.event.inputs.limit != '' }}"
         run: ./hive --client nethermind --sim ${{ github.event.inputs.testSuite }} --sim.limit ${{ github.event.inputs.limit }} --sim.loglevel ${{ github.event.inputs.loglevel }}
-      - name: Upload Hive Results
+      - name: Upload Hive results
         uses: actions/upload-artifact@v2
         with:
           name: results
           path: workspace
           retention-days: 7
-      - name: Move Results
+      - name: Move results
         run: mv workspace /tmp
-      - name: Set Up Python
+      - name: Set up Python
         uses: actions/setup-python@v3
-      - name: Checking out Nethermind
-        uses: actions/checkout@v2        
-      - name: Print Results
-        run: python scripts/testing/hiveTestResultPrinter.py /tmp/workspace/logs/*.json
\ No newline at end of file
+      - name: Check out Nethermind repository
```

Comment:
You could also download the python script with wget instead of checking out the whole repo, might save a few seconds.